src/llvm-late-gc-lowering.cpp

```diff
@@ -1352,7 +1352,7 @@ State LateLowerGCFrame::LocalScan(Function &F) {
                 if (CI->hasStructRetAttr()) {
                     AllocaInst *SRet = dyn_cast<AllocaInst>((CI->arg_begin()[0])->stripInBoundsOffsets());
                     if (SRet) {
```

Comment:
This should perhaps be an assert. There's still lingering problems with this design, but we can keep it for now.